scripts/start/readCheckScenarioConfig.R

```diff
@@ -145,7 +145,7 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
   if (coupling %in% "MAgPIE") {
     knownColumnNames <- c(knownColumnNames, "cm_nash_autoconverge_lastrun", "oldrun", "path_report", "magpie_scen",
                           "no_ghgprices_land_until", "qos", "sbatch", "path_mif_ghgprice_land", "max_iterations",
-                          "magpie_empty", "var_luc")
+                          "magpie_empty", "var_luc","cfg$gms$s15_elastic_demand")
```

Comment:
@dklein-pik Does this work as suggested?

Reply:
Oh, thanks, Flo, this must of cause also be renamed to `cfg_mag$...`